src/coreclr/jit/importer.cpp

```diff
@@ -7256,6 +7256,12 @@ void Compiler::impCheckForPInvokeCall(
 
     if (suppressGCTransition)
     {
+        if (opts.compMCJ)
+        {
+            // MultiCoreJit thread can't inline PInvoke with SuppressGCTransitionAttribute, because it can't be resolved in mcj thread
```

Comment:
```suggestion
            // MultiCoreJit thread can't inline PInvoke with SuppressGCTransitionAttribute, because it can't be resolved
            // in mcj thread
```

To fix formatting requirements